src/main/java/ro/vladutit/Don/t/forget/v2/model/User.java

```diff
@@ -0,0 +1,93 @@
+package ro.vladutit.Don.t.forget.v2.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @NotNull
+    @Column(unique = true)
+    private String username;
+    @NotNull
+    private String password;
+    @NotNull
+    private String fullname;
+    private String telephone;
+    @NotNull
+    private String email;
```

Comment:
email should be unique, right?

Reply:
I do not consider at all that email should always be unique. Initially I only wanted the username to be unique, otherwise there is the possibility to use what they want, how they want, but in the meantime, anyway I decided to make the email unique so that I have a place to contact him, to change them in the future password ... but I could have done the same with the phone number, but let's just say it's not very practical yet.